docs/web-chat/internationalization.mdx

```diff
@@ -100,15 +100,15 @@ window.helpshiftConfig = {
 };
 ```
 
-Alternatively, you can call the `setLanguage` API to set the language after the initialization code.
+Alternatively, you can call the `setLanguage` API (Deprecated) to set the language after the initialization code.
```

Comment:
On Line 98, the comment says "set language to french", but the language key is being set to "de".
Please change it to "fr".